packages/components/src/theme/test/index.tsx

```diff
@@ -45,23 +45,16 @@ describe( 'Theme', () => {
 				inner?.parentElement
 			);
 
-			expect(
-				innerElementStyles.getPropertyValue(
-					'--wp-components-color-accent'
-				)
-			).toBe( '' );
-
-			expect(
-				innerElementStyles.getPropertyValue(
-					'--wp-components-color-accent-darker-10'
-				)
-			).toBe( '' );
-
-			expect(
-				innerElementStyles.getPropertyValue(
-					'--wp-components-color-accent-darker-20'
-				)
-			).toBe( '' );
+			[
+				'--wp-components-color-accent',
+				'--wp-components-color-accent-darker-10',
+				'--wp-components-color-accent-darker-20',
+				'--wp-components-color-accent-inverted',
```

Comment:
Nit: we should probably add a check for `--wp-components-color-accent-inverted` also in the next unit test ("it defines the accent color (and its variations) as a CSS variable")

Reply:
Good catch 🙏